src/xcode/ENA/HealthCertificateToolkit/Tests/HealthCertificateToolkitTests/CertificateAccess/DigitalCovidCertificateAccessTests.swift

```diff
@@ -178,6 +178,30 @@ final class DigitalCovidCertificateAccessTests: XCTestCase {
             XCTAssertEqual(keyIdentifier, expectedKeyIdentifier)
         }
     }
+    
+    func test_When_CBORHasNullValues_Then_SchemaValidationSucceeds() {
```

Comment:
Can we have tests for the nil values are really stripped?
There could be the case, that validation succeeds, result is a success, but still the nil values are not trimmed.